rollup.yaml

```diff
@@ -356,6 +398,12 @@ components:
           example: "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"
           pattern: "^0x([0-9a-fA-F]{40})$"
           format: address
+        value:
+          type: string
+          description: A big-endian 32-byte unsigned integer in hex.
```

Comment:
I think this parameter deserved a better description.
It is the amount of Wei to be passed to the destination.
Most notably, it can be used for Ether withdrawals.